src/utils.ts

```diff
@@ -293,14 +294,47 @@ export function getEnsTextRecord(ens: string, record: string, network = '1') {
   return call(provider, ENS_RESOLVER_ABI, [address, 'text', [ensHash, record]]);
 }
 
-export async function getSpaceUri(id, network = '1') {
+export async function getSpaceUri(
+  id: string,
+  network = '1'
+): Promise<string | null> {
   try {
     return await getEnsTextRecord(id, 'snapshot', network);
   } catch (e) {
-    return false;
+    console.log(e);
+    return null;
   }
 }
 
+export async function getEnsOwner(ens: string): Promise<string | null> {
+  const registryAddress = '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e';
+  const provider = getProvider('1');
+  const ensRegistry = new Contract(
+    registryAddress,
+    'function owner(bytes32) view returns (address)',
+    provider
+  );
+  return await ensRegistry.owner(ens);
```

Comment:
We should pass a ENS like here https://github.com/snapshot-labs/snapshot.js/blob/83f502497f3351d9e6047a047fa4c4aee919cfd0/src/utils.ts#L292 no? 🤔 It works this way? will run a test

Reply:
Yep not working 🤔 It should be like 
```suggestion
  const ensHash = hash(normalize(ens));
  return await ensRegistry.owner(ensHash);
```